pod.go

```diff
@@ -1265,3 +1273,57 @@ func (p *Pod) detachDevices() error {
 
 	return nil
 }
+
+func (p *Pod) updateResources() error {
+	//TODO add support for memory
```

Comment:
Could you raise an issue for this and reference it here?

Reply:
done, Issue: https://github.com/containers/virtcontainers/issues/578

Reply:
Ta - could you add a reference to it in the TODO comment?